<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>materiel</class>
 <widget class="QDialog" name="materiel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建材料</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>材料名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>材料类型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>各向同性材料</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>横观各向同性材料</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>各向异性材料</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>各向同性材料</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <row>
          <property name="text">
           <string>杨氏模量(E)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>泊松比(Nu)</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>横观各向同性材料</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <row>
          <property name="text">
           <string>E1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Nu1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>E2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Nu2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>G2</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>各向异性材料</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget_3">
         <row>
          <property name="text">
           <string>E11</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>E22</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>E33</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Nu12</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Nu23</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Nu13</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>G12</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>G23</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>G13</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>materiel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>materiel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
